packages/generate-bot/templates/cloudbuild-test.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2021 Google LLC
```

Comment:
Can we make these templates all end in `.hbs` if they're handlebars templates? We have magic scripts that look for `cloudbuild.yaml` and `cloudbuild-test.yaml` files and set up automatic Cloud Build triggers for them. If these are committed, they will create deployments and presubmits that will fail.